Nuspec/NGit.nuspec

```diff
@@ -17,10 +17,10 @@
 	</metadata>
 	<files>
 		<file src="Nuspec\RedGate.ThirdParty.NGit.targets" target="build" />
-		<file src="bin\NGit.dll" target="lib" />
-		<file src="bin\NGit.pdb" target="lib" />
+		<file src="bin\netstandard2.0\NGit.dll" target="lib\netstandard2.0\NGit.dll" />
+		<file src="bin\netstandard2.0\NGit.pdb" target="lib\netstandard2.0\NGit.pdb" />
 		<file src="NGit.license.txt" target="App_Readme" />
-		<file src="bin\Sharpen.dll" target="lib" />
-		<file src="bin\Sharpen.pdb" target="lib" />
+		<file src="bin\netstandard2.0\Sharpen.dll" target="lib\netstandard2.0\Sharpen.dll" />
+		<file src="bin\netstandard2.0\Sharpen.pdb" target="lib\netstandard2.0\Sharpen.pdb" />
```

Comment:
Can simplify these to elements like `<file src="bin\netstandard2.0\NGit.*" target="lib\netstandard2.0" />`. This should then also include any xml doc files, which would be a good thing for downstream consumers.